network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/configuration/WgTunnel.kt

```diff
@@ -221,7 +221,6 @@ class RealWgTunnel @Inject constructor(
                     .addAddress(InetNetwork.parse(serverData.address))
                     .apply {
                         addDnsServer(InetAddress.getByName(serverData.gateway))
-                        addDnsServers(netPDefaultConfigProvider.fallbackDns())
```

Comment:
The fallback DNS is always the custom DNS. And it seems the way we handle custom DNS this is never populated when the user configure the DDG default DNS, making this line a noop effectively